ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/Config.scala

```diff
@@ -17,6 +17,13 @@ case class Config(
     jdbcUrl: Option[String] = None,
     expectedUpdatesPath: Option[Path] = None,
     actualUpdatesPath: Option[Path] = None,
+    expectedUpdatesNormalizers: List[UpdateNormalizer] = List(
```

Comment:
Do I have to externalize this with command-line arguments?

Reply:
For coherency I'd say yes, because all other options are, but whether you need it depends on how you're going to use this feature.

Reply:
So far, as you can see [here](https://github.com/DACH-NY/daml-on-kv-oem-integration-kit/pull/1364/files#diff-9a3d122382ec5846024da3a29f7c8da264050a1f66fdfb20821d92d27d0890a3R62). If you don't have a strong opinion we can wait for more comments.